utils/src/main/java/org/jclouds/karaf/utils/EnvHelper.java

```diff
@@ -92,6 +98,22 @@ public static String getComputeCredential(String credential) {
     }
 
     /**
+     * Extracts the credential value from the Google Cloud credentials json file.
+     * @param jsonFile
+     * @return
+     */
+    public static String getGoogleCredentialFromJsonFile(String jsonFile) {
```

Comment:
Do google-cloud-storage and google-compute-engine take credentials in different forms?  I have been using a PEM file and not a JSON-encoded PEM file for my non-CLI testing.